chapters/install/ide-speedup.md

```diff
@@ -0,0 +1,90 @@
+# Speed-up Jina App development in PyCharm and VSCode
+
+Jina App development can be time consuming if you donot have the IDE setup correctly. This chapter describes how to speed up your developer experience
```

Comment:
no need to capitalize app when it's not in the heading. 

do not is two words.